vispy/gloo/glir.py

```diff
@@ -894,13 +898,57 @@ def set_wrapping(self, wrapping):
             gl.glTexParameterf(self._target, GL_TEXTURE_WRAP_R, wrapping[0])
         gl.glTexParameterf(self._target, gl.GL_TEXTURE_WRAP_S, wrapping[-2])
         gl.glTexParameterf(self._target, gl.GL_TEXTURE_WRAP_T, wrapping[-1])
-    
+
     def set_interpolation(self, min, mag):
         self.activate()
         min, mag = as_enum(min), as_enum(mag)
         gl.glTexParameterf(self._target, gl.GL_TEXTURE_MIN_FILTER, min)
         gl.glTexParameterf(self._target, gl.GL_TEXTURE_MAG_FILTER, mag)
 
+# these should be auto generated in _constants.py. But that doesn't seem 
+# to be happening. TODO - figure out why the C parser in (createglapi.py)
+# is not extracting these constanst out.
+# found the constant value at:
```

Comment:
No, not yet. We have a PR open already that does generate them automatically. Right now, our code only populates the constants from the OpenGL ES 2.0 spec. We want to expand it to optionally include those from the desktop spec as well (incl. the 1D and 3D texture constants), but haven't done it yet. If you are feeling adventurous, you can look at #185. We hope to fix it in the next few weeks. In the meantime your hack (following the 3D case) should be sufficient.